convert2rhel/toolopts.py

```diff
@@ -202,7 +202,11 @@ def _register_options(self):
             help="A configuration file to safely provide either a user password or an activation key for registering"
             " the system through subscription-manager. Alternatively, passing these values through the"
             " --activationkey or --password option would leak them through a list of running processes."
-            " Example of this file in /etc/convert2rhel.ini",
+            " Possible locations of the file and it's priority:"
+            " 1) user specified and passed by -c, --config-file option; highest priority"
+            " 2) ~/.convert2rhel.ini; lower priority"
+            " 3) /etc/convert2rhel.ini; the lowest priority"
+            " Example of the file in /etc/convert2rhel.ini",
```

Comment:
I think we should do in the opposite order kind of. Mention that `/etc/convert2rhel.ini` file exists and should be used, but that they can overwrite it by using either `-c|--config-file` or `~/.convert2rhel.ini`

Reply:
@SpyTec Do you mean something like:
```
The example of the file is in /etc/convert2rhel.ini. This file can
be used in this path as the low priority one or in  ~/.convert2rhel.ini
with highest priority. Also is possible to specify different path using 
this option which has the highest priority.
```

Reply:
I think Eric basically just wants the list reversed.  Something like:
```
            " Possible locations of the file and it's priority:"
            " 1) /etc/convert2rhel.ini; systemwide, default values"
            " 2) ~/.convert2rhel.ini; overrides any system defaults with user supplied values"
            " 3) user specified and passed by -c, --config-file option; highest priority, options in this file override anything in (1) and (2) "
```

Reply:
Ok, I understand. I've updated the message to your suggestion and changed the sentence alternatively to Miriam's suggestion.

@SpyTec You can re-review that if it's fine

Reply:
Problem here is though that if I was a new user to convert2rhel reading this I'd assume I have to provide the config file always. So I'd provide `/etc/convert2rhel.ini` even though we already load from there. This should be rewritten a bit to say that the file provided here overwrites the files in etc and user dir and also explain that user dir has higher priority than etc one

Reply:
OK, I'll try to improve that and than consult it with Miriam.

Reply:
@hosekadam consult with whomever is easier or faster for you